certified-connectors/dox42/apiDefinition.swagger.json

```diff
@@ -26,8 +26,8 @@
             "schema": {}
           }
         },
-        "summary": "dox42 Service Call",
-        "description": "Call to the dox42 service",
+        "summary": "dox42 Service Call GET",
+        "description": "Make an HTTP GET request to the dox42 service",
         "operationId": "dox42_Call",
```

Comment:
@dox42diegoschleis operationId properties should [use pascal casing](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#actions-and-triggers). Please make a quick fix for operationId properties.

Reply:
I have changed the operationId to pascal casing